### ISSUE TITLE
Missing aria-label in Bookmark popup

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#3963]
Discovered by @spencerwahl 

Console warning says that the `input` in the bookmark popup needs an aria-label. Maybe important, the `input` is just used to hold the bookmark to copy.
